frontend/src/components/ui/chat/ChatList.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
```

Comment:
Can remove this import, it's not needed haha

Reply:
Just to confirm, there is actually never a need for import React from 'react' in this project right?

Reply:
yes, unless you don't want to `import { useState }` and use `React.useState` in your code